scripts/plot.py

```diff
@@ -68,9 +69,17 @@ def drawLegend(self, legend_list):
     return
 
   def saveImage(self, args):
-    plt.savefig('burndown-' + args.sprint + '.png',bbox_inches='tight')
+    path = args.location
+    filename = 'burndown-' + args.sprint + '.png'
```

Comment:
I would name this variable path_image and add the path to it, so that you don't need the else, so:
```ruby
path = args.location
image_path = 'burndown-' + args.sprint + '.png'

if path
  if not os.path.exists(path):
    os.makedirs(path)
  image_path = os.path.join(path, image_path)

plt.savefig(image_path, bbox_inches='tight')

Reply:
Yeah this cleans up some code, thanks for the suggestion :smile: